DNN Platform/Library/Services/FileSystem/FileInfo.cs

```diff
@@ -218,7 +218,7 @@ public string RelativePath
         }
 
         [XmlElement("size")]
-        public int Size { get; set; }
+        public long Size { get; set; }
```

Comment:
This creates backward compatibility issue. Must deprecate and create another one.